integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/utils/ChromeWebDriverProvider.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.integration.test.web.ui.utils;
+
+import com.datastrato.gravitino.integration.test.util.ITUtils;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.SystemUtils;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.rauschig.jarchivelib.ArchiveFormat;
+import org.rauschig.jarchivelib.Archiver;
+import org.rauschig.jarchivelib.ArchiverFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ChromeWebDriverProvider implements WebDriverProvider {
+  protected static final Logger LOG = LoggerFactory.getLogger(ChromeWebDriverProvider.class);
+  private final String chromeDriverBinName;
+  private final String chromeBinName;
+  private static final String downLoadDir =
+      ITUtils.joinPath(System.getenv("IT_PROJECT_DIR"), "chrome");
+
+  public ChromeWebDriverProvider() {
+    if (SystemUtils.IS_OS_MAC_OSX) {
+      this.chromeDriverBinName = ITUtils.joinPath("chromedriver_mac64", "chromedriver");
+      this.chromeBinName = ITUtils.joinPath("chrome-mac", "Chromium.app");
+    } else if (SystemUtils.IS_OS_LINUX) {
+      this.chromeDriverBinName = ITUtils.joinPath("chromedriver_linux64", "chromedriver");
+      this.chromeBinName = ITUtils.joinPath("chrome-linux", "chrome");
+    } else if (SystemUtils.IS_OS_WINDOWS) {
+      this.chromeDriverBinName = ITUtils.joinPath("chromedriver_win32", "chromedriver.exe");
+      this.chromeBinName = ITUtils.joinPath("chrome-win", "chrome.exe");
+    } else {
+      throw new RuntimeException("Unsupported OS");
+    }
+  }
+
+  @Override
+  public void downloadWebDriver() {
```

Comment:
Why aren't these downloads implemented with a separate script?

Reply:
I think better use JAVA method to download web browser driver.